### DISCUSSION TITLE
Bun v1.1.23

### DISCUSSION THREAD

**Discussion by Jarred-Sumner** [oven-sh/bun#13277] [Announcements]
To install Bun v1.1.23:
```bash
curl -fsSL https://bun.sh/install | bash
# or you can use npm
# npm install -g bun
```

Windows:
```bash
powershell -c "irm bun.sh/install.ps1|iex"
```

To upgrade to Bun v1.1.23:
```bash
bun upgrade
```

### Thanks to 6 contributors!

- [@cirospaciari](https://github.com/cirospaciari)
- [@dylan-conway](https://github.com/dylan-conway)
- [@huseeiin](https://github.com/huseeiin)
- [@Jarred-Sumner](https://github.com/Jarred-Sumner)
- [@nektro](https://github.com/nektro)
- [@paperdave](https://github.com/paperdave)

<hr /><em>This discussion was created from the release <a href='https://github.com/oven-sh/bun/releases/tag/bun-v1.1.23'>Bun v1.1.23</a>.</em>

**Comment by RobinTail** [comment]
@Jarred-Sumner , do you why homebrew does not fetch the new version?

```
$ brew upgrade bun
Warning: oven-sh/bun/bun 1.1.22 already installed
```

**Reply by RobinTail** [reply]
fixed it by running `brew update`
